src/RefractDataStructure.cc

```diff
@@ -489,10 +592,27 @@ namespace drafter {
             }
         }
 
+        template <typename T>
+        struct MakeSectionInfo_ {
```

Comment:
Probably not, It stay there while I testing something and I forgot to remove it.